### PR TITLE
Add `editables` recipe

### PR DESCRIPTION
**Pull request by ofek**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ofek** [comment]
@conda-forge-admin ping conda-forge/help-python please 🙏

File: recipes/editables/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+
+test:
```
Comment (CAM-Gerlach) [inline recipes/editables/meta.yaml:24]: You might want to add the actual test runner deps/command(s) here, assuming it has one.

Comment (ofek) [reply recipes/editables/meta.yaml:24]: The tests are not shipped

Comment (CAM-Gerlach) [reply recipes/editables/meta.yaml:24]: They should be in the `git` tarball used as the basis for the build, are they not?

Comment (dopplershift) [reply recipes/editables/meta.yaml:24]: This recipe is using the PyPI tarball, which doesn't include them. Which is fine. The import check and running `pip check` should be enough to catch any packaging issues.

**Review by dopplershift — APPROVED** [review]

File: recipes/editables/meta.yaml (hunk 1 above)
Comment (ofek) [reply recipes/editables/meta.yaml:24]: The sdist is the source and tests aren't there

**Pull request merged**